File: BiteClub/DTO/Friendship.swift
//
//  Friendship.swift
//  BiteClub
//
//  Created by Ryan Somers on 4/18/25.
//

import Foundation

struct Friendship: Decodable {
    let id: String
    let user_id: String
    let friend_id: String
    let created_at: String
    let updated_at: String
}

File: BiteRunr/shared/src/commonMain/kotlin/org/biterunr_studios/biterunr/DTO/Friendship.kt
package org.biterunr_studios.biterunr.DTO

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class Friendship(
    @SerialName("id")
    val id: String,

    @SerialName("user_id")
    val userId: String,

    @SerialName("friend_id")
    val friendId: String,

    @SerialName("created_at")
    val createdAt: String,

    @SerialName("updated_at")
    val updatedAt: String
)

@Serializable
data class AcceptFriendRequestBody(
    val user_id: String,
    val friend_id: String
)


